src/main/java/uk/co/downthewire/jLTE/fading/ReadFading.java

```diff
@@ -27,12 +27,17 @@ private static void printFadingChannel(int channelId) throws IOException, ClassN
 		Configuration configuration = new PropertiesConfiguration("system.properties").interpolatedConfiguration();
 		String fadingDirectory = configuration.getString(FieldNames.FADING_PATH);
 
-		double seed = 11111.11111;
+		double seed = configuration.getDouble(FieldNames.SEED);
+        int numUEs = configuration.getInt(FieldNames.NUM_UES);
+        int numSectors = 57;
+        int numRBs = configuration.getInt(FieldNames.RBS_PER_SECTOR);
+        int numFadingChannels = numUEs * numSectors * numRBs + 1;
+        double speed = configuration.getDouble(FieldNames.SPEED);
 		int numIterations = 128;
 
 		for (int iteration = 0; iteration < numIterations; iteration++) {
-			String filename = GenerateFading.generateFileName(fadingDirectory, 6555001, 3.0, iteration, seed);
-
+            String filename = GenerateFading.generateFileName(fadingDirectory, numFadingChannels, speed, iteration, seed);
+            System.out.println("filename " + filename);
```

Comment:
I would be inclined to replace this System.out call with a call to the existing LOG object instead.